tests/test_serialization.py

```diff
@@ -274,8 +293,13 @@ def test_incomplete_entry(
                 f.seek(-1, 2)
                 f.truncate()
                 f.seek(0)
-                with pytest.raises(ValueError):
-                    serialization.load_candidate_pairs(f)
+                if load_function is serialization.load_to_iterable:
```

Comment:
I don’t like this branch, but I thought it was the neatest way of doing this. Please let me know if you disagree.

Reply:
I'd probably have written a function that does all the common setup for the test and then have two independent test functions for handling incomplete entries using each loader. This is good too though.

Reply:
If you don’t really care, then I don’t feel like spending any more time on it 🤷🏼‍♂️